### ISSUE TITLE
My first impressions as a Windows user

### ISSUE THREAD

**Issue by Himura2la** [keybase/keybase-issues#2420]
TL;DR: Everything seems to be OK in a starting point. I've managed to import a gpg key, proof twitter and GitHub.
There is a bunch of tiny UI problems although. Starting from the beginning:

- You can add an installer option which adds the keybase install dir to the PATH variable. Searching for the "Keybase Shell" shortcut is not always convenient (yes, Windows users do use command line)
- I didn't get why the system invited me to add a public key without a private key in the very beginning... I haven't manage to follow this way, I had no gpg or keybase installed on the current PC at that time and just copird the public key from mit.edu, My key was on the phone (OpenKeychain) and there's no way to import it from there. Maybe you should add this importing option, the OpenKeychain app has an option to search for keys in keybase.io
- Then I installed gpg and keybase, exported the key from OpenKeychain and imported it to gpg, then added it to keybase. Works fine.

I have managed to catch two UI bugs: 

1. The tray icon doesn't work in unusual multi-screen configuration
![default](https://cloud.githubusercontent.com/assets/7802808/17173434/8bae5392-5403-11e6-9527-5a89af296d59.png)

2. The CLI tool breaks my console color
![2](https://cloud.githubusercontent.com/assets/7802808/17173435/8bb44018-5403-11e6-965c-6ad0aff45a60.png)
![3](https://cloud.githubusercontent.com/assets/7802808/17173464/b30f1aca-5403-11e6-93bf-6ac93c48b308.png)

Thanks for inviting me!

**Comment by exsystemctl** [comment]
I'm having the same issue that you are having on example 1, yet I only have one screen. I can't even use the application after it has been minimized into the tray

**Comment by brandoncurtis** [comment]
Hey @Himura2la, you can add your service verification suggestions to #518.

**Comment by Himura2la** [comment]
Thanks, @brandoncurtis

**Comment by zanderz** [comment]
We have addressed the console colors, that will be fixed in the next update, probably next week.
